file.go

```
@@ -142,32 +140,22 @@ func (f *File) WriteToBuffer() (*bytes.Buffer, error) {
 	zw := zip.NewWriter(buf)
 
 	if err := f.writeToZip(zw); err != nil {
-		return buf, zw.Close()
+		_ = zw.Close()
+		return buf, err
 	}
-
+	if err := zw.Close(); err != nil {
+		return buf, err
+	}
+	f.writeZip64LFH(buf)
```

golangci-lint (1.64.8)

Error return value of `f.writeZip64LFH` is not checked

(errcheck)